### PR TITLE
Enable GitHub Actions for PMM

### PR DESCRIPTION
**Pull request by davidmreed**
GitHub Actions and precommit checks now check and enforce:

- Formatting for Apex, JavaScript, XML, JSON, YAML, Python
- Linting for JavaScript, Python, and Yarn lockfile
- Label audit
- Jest tests

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))
- [ ] Base axe-core JEST test included for any new LWC (No critical violations)
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD/FLS for new object metadata
- [ ] All modified classes are unit tested (Apex) at 100% coverage
- [ ] UX approval or UX not necessary
- [ ] PR contains draft release notes
- [ ] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,5 +6,9 @@
 /tasks/               @SalesforceFoundation/release-engineering-reviewers
 
 # Packaged Metadata
-/force-app/main/default/labels  @microveldt @AndersonTarren
-/force-app/main/default/objects  @microveldt @AndersonTarren
+/force-app/main/default/labels   @AndersonTarren
```
Comment (jjbennett) [inline .github/CODEOWNERS:9]: @AndersonTarren should we also have @lbeller and @smarovich on both of these?

Comment (AndersonTarren) [reply .github/CODEOWNERS:9]: @jjbennett  - I think that's a smart idea.

File: unpackaged/tests/serviceDeliveryModal/lwc/testModal/testModal.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import { LightningElement, wire, track } from "lwc";
 import { CurrentPageReference } from "lightning/navigation";
-import { fireEvent } from "c/pubsub";
-import { debug, debouncify } from "c/util";
+import { debug } from "c/util";
```
Comment (jjbennett) [inline unpackaged/tests/serviceDeliveryModal/lwc/testModal/testModal.js:3]: @AndersonTarren - do we still need this?

Comment (AndersonTarren) [reply unpackaged/tests/serviceDeliveryModal/lwc/testModal/testModal.js:3]: I believe this component was initially created to use in testing the CaseMan modal directly from PMM.  If we don't feel like it's being used in that manner, or we might not use it in the future, I could see getting rid of it.

File: .github/CODEOWNERS (hunk 1 above)
Comment (davidmreed) [reply .github/CODEOWNERS]: Done

**Pull request merged**